dist2src/worker/processor.py

```diff
@@ -100,5 +100,7 @@ def process_message(self, event: dict, **kwargs):
         d2s.convert(branch, branch)
 
         # Push the result to source-git.
+        src_git_repo.git.push("origin", branch)
+
         # Update moves sg-start tag, we need --tags --force to move it in remote.
-        src_git_repo.git.push("origin", branch, tags=True, force=True)
+        src_git_repo.git.push(tags=True, force=True)
```

Comment:
I'm not sure about this. I'd leave it as it is.

Reply:
But then it will fail, isn't it?

Reply:
https://github.com/packit/source-git-onboarding/pull/5/commits/ef0071594a41fe6f983a09b2023ff260d0a9672e worked for me in https://git.stg.centos.org/source-git/adobe-mappings-cmap/commits/c8

Reply:
Good point, though I think the branch can be skipped (doesn't need to be pushed the second time). I've updated it to `src_git_repo.git.push("origin", tags=True, force=True)` to have the remote explicit.